### DISCUSSION TITLE
a button to refresh the page manually?

### DISCUSSION THREAD

**Discussion by fivestone** [trutzig89182/Mastodon-Sociologists#40] [Ideas]
hi, I know little about webpage frontend. I just wonder that the list generated through app.js may not refresh for a long time in the browser's cache after the new records are added to csv. So, is it necessary to add a button to refresh the list manually?

**Comment by trutzig89182** [comment]
Hey, 

I probably don’t know more about that then you do. I my experience the page did upload when opened. 

If you feel there is a need for that, you could however have a go. I for my part won’t work on that. I will have to see if I am able to develop this further or just updat the csv. But if I find he time I would rather focus on making filtering options based on keywords and languages.

**Comment by trutzig89182** [comment]
Ok, just now I had the problem you described. But I still wonder if it’s a problem for the usual user who just visits the page once in a while. What do you think, @fivestone ?
